game/games_test.go

```diff
@@ -1,14 +1,76 @@
 package game
 
 import (
+	"database/sql"
 	_ "encoding/json"
 	"net/http"
 	"net/http/httptest"
+	"os"
 	"testing"
+
+	_ "github.com/mattn/go-sqlite3"
 )
 
+func checkErr(t *testing.T, err error) {
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+func prepareFromFile(db *sql.DB, path string) (*sql.Stmt, error) {
+	data, err := os.ReadFile(path)
+	if err != nil {
+		return nil, err
+	}
+
+	stmt, err := db.Prepare(string(data))
+	if err != nil {
+		return nil, err
+	}
+
+	return stmt, nil
+}
+
+func dbSetup(t *testing.T) *sql.DB {
+	db, err := sql.Open("sqlite3", ":memory:")
+	checkErr(t, err)
+
+	stmt, err := prepareFromFile(db, "../db/create_games.sql")
+	checkErr(t, err)
+
+	_, err = stmt.Exec()
+	checkErr(t, err)
+
+	stmt, err = prepareFromFile(db, "../db/seed_games.sql")
+	checkErr(t, err)
+
+	_, err = stmt.Exec()
+	checkErr(t, err)
+
+	return db
+}
+
+func TestAll(t *testing.T) {
+	db := dbSetup(t)
+  defer db.Close()
+
+	games, err := All(db)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if count := len(games); count < 1 {
```

Comment:
shouldn't it be `count != 2`

Reply:
Since both seeds were in one file, the prepare statement would only load the first one. I've split it now, load them separately, and check for both.